Lib/test/test_traceback.py

```diff
@@ -16,8 +16,9 @@
 
 
 test_code = namedtuple('code', ['co_filename', 'co_name'])
+test_code.co_positions = lambda _: [(6, 6, 0, 0)]
```

Comment:
I think it would be better if we imitate the actual behavior (returning an iterator instead of a list)

Reply:
Good point. Wrapped it in an `iter()` call, is that better?